static/js/brand-store/Settings/Settings.tsx

```diff
@@ -53,12 +53,13 @@ function Settings() {
   const [isSaving, setIsSaving] = useState(false);
   const [showSuccessNotification, setShowSuccessNotification] = useState(false);
   const [showErrorNotification, setShowErrorNotification] = useState(false);
-  const [currentMember, setCurrentMember] = useState<CurrentMember>({
+  const [currentMember, setCurrentMember] = useState<Member | undefined>({
```

Comment:
It's a bit weird that this can be undefined or a Member, but you start it out with an empty dataset which is kind of neither...I'd prefer if this was just undefined to start off.
```suggestion
  const [currentMember, setCurrentMember] = useState<Member | undefined>();
```